prow/config/config.yaml

```diff
@@ -727,7 +716,7 @@ presubmits:
         image: quay.io/metal3-io/capm3-unit:main
         imagePullPolicy: Always
   - name: manifestlint
-    always_run: true
+    skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
```

Comment:
probably including .go also makes sense here, because it checks only YAML files. Or we could be more explicit with  `run_if_changed:` to run only when YAML/YML files are touched.

Reply:
Thanks, I wasn't sure if that would catch everything or not.
One slight worry is that ideally we should run the job if the Makefile or the script it is running changes, and setting that up is ugly given the terrible syntax of these config options.

Reply:
yeah, regex makes it hard to read :(